### PR TITLE
Update description and format, fix typos

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

- Update supported TiDB version from `v4.0.1` to `v4.0.5`
- Change past tense verbs to root form verbs in release notes
- Update anchor links in `TOC.md` to allow users see the very beginning of one doc
- Make table alignment consistent (left-aligned)
- Add code snippets for copying code blocks
- Update format of a list within a Note
- Fix several typos

### What is the related PR or file link(s)?

Reference link(s):  N/A

### CODE REVIEW

File: limitations-and-quotas.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ summary: Learn the limitations and quotas in TiDB Cloud.
 
 # Limitations and Quotas in TiDB Cloud
 
-TiDB Cloud limits how many of each kind of component you can create, and the common usage limitations of TiDB. In addition, there are some orgnization-level quotas to limit the amount of resources created by users to prevent from creating more resources than you actually need.
-These tables outline limits and quotas. 
+TiDB Cloud limits how many of each kind of component you can create, and the common usage limitations of TiDB. In addition, there are some organization-level quotas to limit the amount of resources created by users to prevent from creating more resources than you actually need. These tables outline limits and quotas.
```
Comment (lilin90) [inline limitations-and-quotas.md:8]: FYI. Only fixed one typo here: orgnization > organization

File: terms-of-service.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ PLEASE READ THESE TERMS AND CONDITIONS CAREFULLY BEFORE USING THE TIDB CLOUD SER
...
+YOU WARRANT THAT YOU ARE AT LEAST EIGHTEEN YEARS OLD AND THAT YOU HAVE THE LEGAL CAPACITY TO ENTER INTO CONTRACTS. IF YOU ARE ENTERING INTO THIS AGREEMENT ON BEHALF OF A COMPANY, ORGANIZATION, OTHER ENTITY OR THEIR AFFILIATES, THEN (A) "YOU" AS USED HEREIN ADDITIONALLY REFERS TO THAT COMPANY, ORGANIZATION, ENTITY OR THEIR AFFILIATES; (B) YOU REPRESENT AND WARRANT THAT YOU HAVE THE AUTHORITY TO BIND SUCH COMPANY, ORGANIZATION, OR ENTITY TO THE TERMS AND CONDITIONS SET FORTH HEREIN; (C) NO AUTHORIZATION OR APPROVAL FROM ANY THIRD PARTY IS REQUIRED IN CONNECTION WITH YOUR EXECUTION, DELIVERY OR PERFORMANCE OF THIS AGREEMENT; AND (D) THE EXECUTION, DELIVERY AND PERFORMANCE OF THE AGREEMENT DOES NOT AND WILL NOT VIOLATE THE TERMS OR CONDITIONS OF ANY OTHER AGREEMENT TO WHICH YOU ARE A PARTY OR BY WHICH IT IS OTHERWISE BOUND, OR ANY APPLICABLE LAW OR REGULATION. **IF YOU DO NOT AGREE TO ANY TERMS OF THIS AGREEMENT OR IF YOU DO NOT HAVE SUCH AUTHORITY TO EXECUTE THIS AGREEMENT, YOU MUST IMMEDIATELY CEASE USING THE TIDB CLOUD SERVICES.**
```
Comment (lilin90) [inline terms-of-service.md:12]: Only fixed typo: AFFLIATES > AFFILIATES

File: connect-to-tidb-cluster.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ To connect to your TiDB cluster using a SQL client, perform the following steps:
 
 4. Use a SQL client to connect to TiDB from your server which has set up VPC peering with TiDB Cloud.
 
+    {{< copyable "sql" >}}
```
Comment (tennix) [inline connect-to-tidb-cluster.md:25]: This is a shell command
```suggestion
    {{< copyable "shell" >}}
```

File: connect-to-tidb-cluster.md [hunk 4]
```diff
@@ -22,6 +22,8 @@ To connect to your TiDB cluster using a SQL client, perform the following steps:
...
+
     ```sql
```
Comment (lilin90) [inline connect-to-tidb-cluster.md:27]:
```suggestion
    ```shell
```

Comment (tennix) [inline connect-to-tidb-cluster.md]:
```suggestion
    ```shell
```

File: migrate-data-into-tidb.md [hunk 5]
```diff
@@ -49,14 +54,18 @@ You can use [TiDB Lightning](https://docs.pingcap.com/tidb/v4.0/tidb-lightning-o
 
 1. Install TiDB Lightning:
 
-    ```
+    {{< copyable "shell-regular" >}}
+
+    ```bash
```
Comment (tennix) [inline migrate-data-into-tidb.md:59]: Use `shell` to keep consistent?

Comment (lilin90) [reply migrate-data-into-tidb.md]: Sounds good. Resolved.

**Review by tennix — APPROVED** [review]
LGTM

**Pull request merged**